web/api/v1/api.go

```diff
@@ -476,6 +480,11 @@ func (api *API) query(r *http.Request) (result apiFuncResult) {
 		return apiFuncResult{nil, returnAPIError(res.Err), res.Warnings, qry.Close}
 	}
 
+	var warnings annotations.Annotations
```

Comment:
```suggestion
	warnings := res.Warnings
```

We should not disregard the warnings coming from the query.